packages/babel-helpers/src/helpers.js

```diff
@@ -442,6 +442,23 @@ helpers.specRequireInterop = template(`
   })
 `);
 
+helpers.specImportCheck = template(`
+  (function (module, imports) {
+    if (!module.__esModule) throw new Error("Only ES modules can be checked");
```

Comment:
This means you can only `import` things that babel produced - that doesn't seem right *at all*. the "spec" option is useless if it forbids you from importing CJS modules.